cmd/ak/cmd/vars/set.go

```diff
@@ -44,7 +44,7 @@ var setCmd = common.StandardCommand(&cobra.Command{
 			value = string(data)
 		}
 
-		ev := sdktypes.NewVar(n, value, secret).WithScopeID(id)
+		ev := sdktypes.NewVar(n).SetValue(value).SetSecret(secret).SetRequired(required).WithScopeID(id)
```

Comment:
I don't like this style of coding, prefer the former way.

Reply:
it's actually make it easier to do stuff - not all are secrets, not all have values, etc.